packages/template-chakra-storefront/src/hooks/use-auth-modal.test.js

```diff
@@ -310,7 +309,7 @@ test('Renders error when given incorrect log in credentials', async () => {
             expect(screen.getByText(/Something went wrong. Try again!/i)).toBeInTheDocument()
         },
         {
-            timeout: 10000
+            timeout: 1000
```

Comment:
1 second is pretty short and it feels like you've fixed the test so we no longer need the hacky timeouts, maybe we can just remove the timeout option and use the default? Does `timeout` have to be specified for this unit test to work?